terraform/main.tf

```
@@ -23,11 +23,11 @@ resource "lxd_network" "sdcore-mgmt" {
     "ipv4.address" = "10.201.0.1/24"
     "ipv4.nat"     = "true"
     "ipv6.address" = "none"
-    "dns.mode" = "managed"
-    "dns.domain" = "mgmt"
-    "raw.dnsmasq" = <<-EOF
+    "dns.mode"     = "managed"
+    "dns.domain"   = "mgmt"
```

I think the dns domain should be changed
```suggestion
    "dns.domain"   = "mgmt.local"
```

And similarly, the amf record should be updated at the same time to ensure the DNS does not get confused.

---

Agreed, done